.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = tab
```

Comment:
Should I do a huge "Replace in project" then replacing all tabs with 2 (or 4?) spaces?